apps/site/lib/site_web/templates/schedule/_cms_teasers.html.eex

```diff
@@ -1,3 +1,3 @@
-<%= if @featured_content do SiteWeb.PartialView.teaser(@featured_content, class: "m-schedule-line__teaser") end %>
+<%= if @featured_content do SiteWeb.PartialView.teaser(@featured_content, class: "m-schedule-page__teaser") end %>
```

Comment:
I'm not sure i understand why we're separating these into two templates just to use different classes. is there harm in just updating the class in the existing template?

Reply:
The schedule note is going at the top of the column in the new page instead of the teasers but the schedule note doesn't exist on the old page. The other template can be deleted once we remove the flag (the selector is specific to the schedule page).

Reply:
I can move the classnames as an option to the render instead. It's a margin change for both desktop and mobile.